internal/db/schema/migrations/oss/postgres/19/03_session.up.sql

```diff
@@ -0,0 +1,6 @@
+begin;
+
+alter table session_connection
+  add column user_client_ip inet;
```

Comment:
Can a `not null` constraint be added? For existing rows, I suggest setting the value to `::` (the [IPv6 Unspecified Address](https://en.wikipedia.org/wiki/IPv6_address#Unspecified_address)).

Reply:
ty!